src/guiguts/application.py

```diff
@@ -74,7 +79,7 @@ def open_document(self, args):
 
         Accepts a list of filenames, but only loads the first.
         """
-        file.load_file(args[0])
+        self.file.load_file(args[0])
```

Comment:
This is an untested change but I think this is correct as `file.py` does not have a `load_file()` function -- the `File` class does though.

Reply:
Thanks for catching that